packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -42,4 +72,70 @@ describe('Error', () => {
         // Assert
         expect(description.text()).toMatchSnapshot();
     });
+
+    describe('methods', () => {
+        describe('`redirectToMenu` ::', () => {
+            const oldWindowLocation = window.location;
+
+            beforeAll(() => {
+                delete window.location;
+
+                window.location = Object.defineProperties(
+                    {},
+                    {
+                        ...Object.getOwnPropertyDescriptors(oldWindowLocation),
+                        assign: {
+                            configurable: true,
+                            value: jest.fn()
+                        }
+                    }
+                );
+            });
+
+            afterAll(() => {
+                window.location = oldWindowLocation;
+            });
```

Comment:
This seems quite complicated to configure the spy for `window.location.assign`.
We have a few examples in `Checkout.test` already, but in a nutshell, this should do it:

```
const windowLocationSpy = jest.spyOn(window.location, 'assign').mockImplementation();

// Do something

expect(windowLocationSpy).toHaveBeenCalledWith('...');

Reply:
This won't work, originally did this and it failed.